web/documentserver-example/java/src/main/java/format/Format.java

```diff
@@ -0,0 +1,69 @@
+/**
+ *
+ * (c) Copyright Ascensio System SIA 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package format;
+
+import java.util.List;
+
+public final class Format {
+    private final String name;
+
+    public String getName() {
+        return this.name;
+    }
+
+    private final String type;
```

Comment:
The Type field can be made of type [FileType ](https://github.com/ONLYOFFICE/document-server-integration/blob/feature/java-matrix-formats/web/documentserver-example/java/src/main/java/entities/FileType.java)